uniprotkb-rest/src/main/java/org/uniprot/api/uniprotkb/service/UniProtSolrSortClause.java

```diff
@@ -0,0 +1,33 @@
+package org.uniprot.api.uniprotkb.service;
+
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import org.uniprot.api.rest.search.AbstractSolrSortClause;
+import org.uniprot.store.search.field.UniProtField;
+
+@Service
+public class UniProtSolrSortClause extends AbstractSolrSortClause {
```

Comment:
Here I recommend you request Edd Review, because the Uniprot Sort/Rank logic is complex, and he knows more about it and would be able to better validate it

Reply:
I have requested @eddturner to review all the changes.